ci.sh

```diff
@@ -360,12 +360,12 @@ if [ "$PROVIDER_NAME" = "cargo-check" ]; then
     # - openSUSE Tumbleweed
     # - openSUSE Leap 15.4
 
-    rustup toolchain install ${MSRV}
+    rustup default ${MSRV}
     # TODO: The "jwt-svid-authenticator" is currently not being used.
     RUST_BACKTRACE=1 cargo +${MSRV} check --release --features=all-providers,direct-authenticator,unix-peer-credentials-authenticator
```

Comment:
We use `cargo +${MSRV} check ...` or `cargo +stable check ..` to consider particular toolchain. So do we need this?